components/worktemplates/index.tsx

```diff
@@ -0,0 +1,170 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState} from 'react';
+import classnames from 'classnames';
+import {useIntl} from 'react-intl';
+import {useDispatch} from 'react-redux';
+import styled from 'styled-components';
+
+import {Visibility, WorkTemplate} from '@mattermost/types/worktemplates';
+import {closeModal as closeModalAction} from 'actions/views/modals';
+import LocalizedIcon from 'components/localized_icon';
```

Comment:
nit: group imports by context

Reply:
Not sure what do you mean by context?

Reply:
all the ones form 'components' in one group, the ones from '@mattermost' another group, separated by blank lines, again, this is a nit ;)

Reply:
We have an ESLint configuration that does some amount of this, but we really need to update it at some point to do the rest of that for us